index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main-grid">
+        <section class="navigation">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/X_logo_2023_%28white%29.png/900px-X_logo_2023_%28white%29.png">
+        <h4>Home</h4>
+        <h4>Explore</h4>
+        <h4>Notifications</h4>
+        <h4>Messages</h4>
```

Comment:
I see the use of h4 but I'm wondering if buttons would be a more appropriate element to use to allow it to be selected/tab-able. I'm imagining that you might of modified the h4 to allow for this possibly.